codechecker_lib/analyzers/result_handler_base.py

```diff
@@ -183,17 +183,24 @@ def get_analyzer_result_file(self):
                 '_' + analyzed_file_name + '_' + uid + '.plist'
 
             out_file = os.path.join(self.__workspace, out_file_name)
-            self.__res_file = out_file
+            self.__result_file = out_file
+
+        return self.__result_file
 
-        return self.__res_file
+    @analyzer_result_file.setter
```

Comment:
There was no setter for the result file so it should not be possible to accidentally overwrite during the analysis process the result file. The cleanup might fail to remove temporary result files.